src/providers/https.ts

```diff
@@ -77,6 +82,19 @@ export function _onRequestWithOptions(
     convertInvoker
   );
   // TODO parse the options
+
+  cloudFunction.__endpoint = {
+    platform: 'gcfv1',
+    ...optionsToEndpoint(options),
+    httpsTrigger: {},
+  };
+  convertIfPresent(
+    cloudFunction.__endpoint.httpsTrigger,
+    options,
+    'invoker',
+    'invoker',
+    convertInvoker
```

Comment:
doesn't `convertInvoker` turn `foo@` into `foo@project.gserviceaccount.com` and we didn't want that?

Reply:
We have that kind of logic for service accounts:

https://github.com/firebase/firebase-functions/blob/38f1946df4804002be9c585f386c5ae61fd45b10/src/common/encoding.ts#L53-L72

But not for invoker - we only do light validation checks, e.g. to make sure that invoker option with `public` or `private` doesn't conflict, etc.

Reply:
Let's keep the shorthand syntax in the manifest so we can stay project-independent.